src/config/mod.rs

```diff
@@ -2624,9 +2624,9 @@ impl Default for BackupStreamConfig {
         let total_mem = SysQuota::memory_limit_in_bytes();
         let quota_size = (total_mem as f64 * 0.1).min(ReadableSize::mb(512).0 as _);
         Self {
-            max_flush_interval: ReadableDuration::minutes(3),
+            max_flush_interval: ReadableDuration::secs(150),
             // use at most 50% of vCPU by default
-            num_threads: (cpu_num * 0.5).clamp(2.0, 12.0) as usize,
+            num_threads: cpu_num.clamp(2.0, 16.0) as usize,
```

Comment:
How much actual CPU will be used by backup after this change? Will it increase the IO usage as well?

Reply:
- The old code: num-threads = sys_cpu_num * 0.5 
    sys_cpu_num/4 used to scan kvs when met new region-leader;
    sys_cpu_num/4 used to backup log
- The code in pr: num-threads = sys_cpu_num, and limited at [2, 16]

Reply:
If less the count of kvs need backup/scan, the threads will on the free status. 
If a large mount of kvs need to backup/scan,  the increasing the num-threads will enhance the RPO in PITR.